src/test/hello.test.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { expect } from "chai";
+
+describe("Query hello", () => {
+  const EXPECTED_RESPONSE = { data: { hello: "Hello Word!" } };
+  it("should return hello word", async () => {
```

Comment:
aqui dentro do it, a gente coloca o teste e tenta seguir um padrão chamado AAA
Arrange - ajusta o que precisar (cria seed do banco, input da query/mutation, etc)
Action - ação: coisas que vc precisa fazer (chamada da query/mutation, criar expect response, buscar valor atualizado no banco, etc)
Assert - verificações/expects